execution/src/execution.rs

```diff
@@ -220,7 +220,7 @@ impl<R: ExecutionRpc> ExecutionClient<R> {
         let tx_hashes = payload
             .transactions
             .iter()
-            .map(|tx| H256::from_slice(&keccak256(tx.to_vec())))
+            .map(|tx| keccak256(tx.deref()).into())
```

Comment:
Good! But still has one more issue.

I just give you a tip, not the solution.

Read the following references:
- [More on Deref coercion](https://doc.rust-lang.org/std/ops/trait.Deref.html#more-on-deref-coercion)
- [The Rust Reference: Coercion types](https://doc.rust-lang.org/reference/type-coercions.html#coercion-types)

Reply:
Try running `cargo build` after using `&x` to instead of `x.deref()` in your PR.

Then `cargo` will tell you how to write the code.